configure.ac

```diff
@@ -2314,6 +2314,13 @@ AC_CHECK_HEADERS(linux/can.h linux/can/bcm.h linux/can/j1939.h linux/can/raw.h,,
 #endif
 ])
 
+# CAN protocol constants on NetBSD are defined in this header
+AC_CHECK_HEADERS(netcan/can.h,,,[
+#ifdef HAVE_SYS_SOCKET_H
+#include <sys/socket.h>
+#endif
```

Comment:
Do you really need to include `<sys/socket.h>` for the header check?

Reply:
not anymore, check done earlier already anyway.